cms/static/js/views/utils/view_utils.js

```diff
@@ -173,6 +174,33 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
             return false;
         };
 
+        keywordValidator = (function () {
+            var regexp = /%%+[^%]+%%/g;
```

Comment:
Good question / catch, I took this code verbatim from Nick's course announcement branch :P I'll fix that.